### PR TITLE
Housekeeping

### PR DESCRIPTION
**Pull request by naszam**
- [x] Add oracles goerli deployer + skip wards
- [x] Add circuit breakers + minor change in `get-created-timestamp.sh`
- [x] Add ci test name
- [x] Update readme
- [x] Test PASS

### CODE REVIEW

**Comment by Geoge1** [comment]
Kindly open this live chat link to talk to the customer service directly on your issue  https://direct.lc.chat/14164218/

File: README.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ If the weekly executive needs to onboard a new collateral:
 $ make
 ```
 
-### Test (DappTools with Optimizations)
+### Test (DappTools without Optimizations)
```
Comment (brianmcmichael) [inline README.md:30]: Optimizer should default to `on` with the `make test`

https://github.com/makerdao/spells-goerli/blob/PE-1004-20220527/scripts/test-dssspell.sh#L20

Comment (naszam) [reply README.md:30]: that looks currently commented/disabled

Comment (naszam) [reply README.md:30]: Considering that `make test-dev` is also currently present (where `optimizer` is set to zero), we could consider to keep `make test` optimizer on by default and uncomment the export block.

Comment (gbalabasquer) [reply README.md:30]: We stopped using the optimizer due the time taking to compile. But honestly I think that delay was with 1 run and wasn't happening with the default 200 runs. No idea why.
We should double check it and if actually works fine with 200 runs renable it here and on mainnet.

Comment (brianmcmichael) [reply README.md:30]: Maybe we can leave those blocks uncommented and set the default optimizer state to off, then.

IIRC this was a solc issue and not a dapp issue. Not sure if we're still experiencing it with foundry though.

Comment (naszam) [reply README.md:30]: Done in `3a83e91`

File: Makefile [hunk 2]
```diff
@@ -3,8 +3,8 @@ all                :; DAPP_LIBRARIES=' lib/dss-exec-lib/src/DssExecLib.sol:DssEx
                        dapp --use solc:0.6.12 build
 clean              :; dapp clean
                       # Usage example: make test match=SpellIsCast
-test               :; ./scripts/test-dssspell.sh match="$(match)" optimizer="$(optimizer)" block="$(block)"
-test-dev           :; ./scripts/test-dssspell.sh match="$(match)" optimizer="0" block="$(block)"
+test               :; ./scripts/test-dssspell.sh match="$(match)" optimizer="$(optimizer)" optimizer-runs="$(optimizer-runs)" block="$(block)"
+test-dev           :; ./scripts/test-dssspell.sh match="$(match)" optimizer="1" optimizer-runs="$(optimizer-runs)" block="$(block)"
```
[the remark this answers is not in the document]
Comment (gbalabasquer) [reply Makefile:7]: Why do we want to have `test-dev` if we are enabling it everywhere?
`test-dev` was created for the moments where we still use optimizations on for deployment. So we wanted to have a quick path for testing during development until deployment time.

[the remark this answers is not in the document]
Comment (gbalabasquer) [reply Makefile:7]: btw did you check if with optimizations on and runs 200 the compilation time is fast?
As if that is the case we should just use that configuration everywhere and stop all these discussions.

[the remark this answers is not in the document]
Comment (naszam) [reply Makefile:7]: I've just double checked and I don't see any noticeable difference with optimizations on and 200 runs in terms of compilation time.

[the remark this answers is not in the document]
Comment (gbalabasquer) [reply Makefile:7]: any difference against optimizations off or against optimizations on and 1 run?

[the remark this answers is not in the document]
Comment (naszam) [reply Makefile:7]: sorry, I meant against optimizations on and 1 runs

[the remark this answers is not in the document]
Comment (gbalabasquer) [reply Makefile:7]: Ahh ok, so still takes time to compile with optimizations on and 200 runs.

[the remark this answers is not in the document]
Comment (naszam) [reply Makefile:7]: yep

[the remark this answers is not in the document]
Comment (naszam) [reply Makefile:7]: it actually takes more time to compile, ~30 min vs ~20 min with 1 run

[the remark this answers is not in the document]
Comment (naszam) [reply Makefile]: as discussed in chat, I've removed `test-dev` Makefile script as we're going with optimization off by default.

File: Makefile [hunk 3]
```diff
@@ -3,8 +3,7 @@ all                :; DAPP_LIBRARIES=' lib/dss-exec-lib/src/DssExecLib.sol:DssEx
                        dapp --use solc:0.6.12 build
 clean              :; dapp clean
                       # Usage example: make test match=SpellIsCast
-test               :; ./scripts/test-dssspell.sh match="$(match)" optimizer="$(optimizer)" block="$(block)"
-test-dev           :; ./scripts/test-dssspell.sh match="$(match)" optimizer="0" block="$(block)"
+test               :; ./scripts/test-dssspell.sh match="$(match)" optimizer="$(optimizer)" optimizer-runs="$(optimizer-runs)" block="$(block)"
 test-forge         :; ./scripts/test-dssspell-forge.sh match="$(match)" block="$(block)"
 estimate           :; ./scripts/estimate-deploy-gas.sh
 deploy             :; make && dapp create DssSpell | xargs ./scripts/verify.py DssSpell
```
Comment (brianmcmichael) [inline Makefile:9]: It looks like calling `make deploy` will deploy the contract with optimizations off. Is that where we want to be here?

Comment (naszam) [reply Makefile:9]: It seems that general consensus emerged so far is leaning towards testing and deploying spells without optimizations to speed up the process and reduce overhead.

**Review by brianmcmichael — APPROVED** [review]
Still not sold on deploying un-optimized, but it's probably safer and we can revisit later if necessary.

LGTM

**Pull request merged**